src/ui/ui.css

```diff
@@ -23,8 +23,7 @@ body {
   position: absolute;
   left: 0;
   top: 0;
-  padding: 16px;
-  margin: 0;
+  margin: 16px;
```

Comment:
In firefox, when `padding: 16px;`, the Close button does not show with 16px padding on left.

Reply:
The padding was used not for positioning, but to enlarge the tap target IIRC. Margin doesn't give us bigger targets.